homework_10/FileLogger.java

```diff
@@ -0,0 +1,61 @@
+import enums.LoggingLevel;
+import exceptions.FileMaxSizeException;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileLogger {
+    private final FileLoggerConfiguration config;
+
+    public FileLogger(FileLoggerConfiguration config) {
+        this.config = config;
+    }
+
+    public void debug(String message) {
+        if (isDebugEnabled()) {
+            log(message, LoggingLevel.DEBUG);
+            log(message, LoggingLevel.INFO);
```

Comment:
No, sorry, but this is wrong. You should call `log` only once here, but not twice!